### PR TITLE
Finished Homework2

### PR DESCRIPTION
**Pull request by regina-ctrl**

### CODE REVIEW

File: Lesson2/src/Homework.java [hunk 1]
```diff
@@ -27,6 +27,10 @@ private static boolean isRacecar(int numWheels, double mpg, boolean isManual) {
         //   1) The vehicle must have exactly 4 wheels.
         //   2) The vehicle must get 13 mpg or less; race cars are inefficient AF! NOTE: This value can't be zero ornegative!
         //   3) The vechiel must be a manual.
+
+        if (numWheels==4 && mpg>0 && mpg<=13 && isManual==true ) {
```
Comment (Palidor) [inline Lesson2/src/Homework.java]: Looks great! Couple things that i'm going to nit pick for the purposes of going through the process here.

With regards to formatting... 

1. When you use a comparator operator such as == or >, you want to have a space after the variable and before the value, e.g. "numWheels == 4". 
2. There's an extra space after "true" at the end of the if block, e.g. "isManual == true)".

**Review by Palidor — CHANGES_REQUESTED** [review]

File: Lesson2/src/Homework.java [hunk 2]
```diff
@@ -28,7 +28,7 @@ private static boolean isRacecar(int numWheels, double mpg, boolean isManual) {
         //   2) The vehicle must get 13 mpg or less; race cars are inefficient AF! NOTE: This value can't be zero ornegative!
         //   3) The vechiel must be a manual.
 
-        if (numWheels==4 && mpg>0 && mpg<=13 && isManual==true ) {
+        if (numWheels == 4 && mpg > 0 && mpg <= 13 && isManual == true ) {
```
Comment (Palidor) [inline Lesson2/src/Homework.java]: Good enough lol

**Review by Palidor — APPROVED** [review]

File: Lesson2/src/Homework.java [hunk 3]
```diff
@@ -28,7 +28,7 @@ private static boolean isRacecar(int numWheels, double mpg, boolean isManual) {
         //   2) The vehicle must get 13 mpg or less; race cars are inefficient AF! NOTE: This value can't be zero ornegative!
         //   3) The vechiel must be a manual.
 
-        if (numWheels==4 && mpg>0 && mpg<=13 && isManual==true ) {
+        if (numWheels == 4 && mpg > 0 && mpg <= 13 && isManual == true) {
```
Comment (Palidor) [inline Lesson2/src/Homework.java:31]: Ok fine, smart ass.

**Review by Palidor — APPROVED** [review]

File: Lesson2/src/Homework.java (hunk 3 above)
Comment (regina-ctrl) [reply Lesson2/src/Homework.java:31]: you are a smart ass

**Pull request merged**